apps/aechannel/test/aesc_txs_SUITE.erl

```diff
@@ -5230,3 +5280,46 @@ compile_contract(ContractName) ->
     aect_test_utils:compile_contract(
       filename:join(["contracts",
                      filename:basename(ContractName, ".aes") ++ ".aes"])).
+
+%% test that a force progress transaction can NOT produce an on-chain
+%% contract with a code with the wrong serialization
+fp_wrong_encoded_code(Cfg) ->
```

Comment:
Could you please confirm that running this test having reverted [`check_code_serialization` change in `check_force_progress_`](https://github.com/aeternity/aeternity/pull/2146/commits/5630cc259b21dc91f47d607050098ee66c6ce4e9) fails.

Reply:
Confirmed. It fails if the code is not fixed.